core/src/main/scala/za/co/absa/fadb/status/handling/implementations/UserDefinedStatusHandling.scala

```diff
@@ -1,3 +1,5 @@
+package za.co.absa.fadb.status.handling.implementations
```

Comment:
I think that the licence should go first

Reply:
thanks, I have put the license on top ...